.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: ci
+
+on:
+  push:
+    branches: ['main']
```

Comment:
This is a bit ambiguous, as the CI is already run on every PR to main

Reply:
I know, but it can be quite useful when merging multiple branches around the same time - at least in my experience. 
This is a "bewuste" double check.

Reply:
Alright!